### ISSUE TITLE
[Feature Request] TuMangaOnline : Choose Different Scans

### ISSUE THREAD

**Issue by MerradOklom** [inorichi/tachiyomi-extensions#1671]
### What feature should be added to this source?

There are some manga which are translated by different scans but currently you can only read from one of them, that is whoever uploaded the chapter first. Take for example this manga:
https://tmofans.com/library/manga/13090/Smokin-Parade
Chapter 10 was uploaded by Gotei Konoha on 15-09-2016 and later by Raven Scans on 08-12-2016, but in the app you can only read the one from Gotei Konoha. 

Tl;dr the possibility to select chapters by different scans
